Arrowgene.Ddon.GameServer/Characters/QuestManager.cs

```diff
@@ -146,19 +162,20 @@ public static Quest GetQuestByScheduleId(uint questScheduleId)
             return gQuests[questScheduleId];
         }
 
-        public static List<Quest> GetQuestsByQuestId(QuestId questId)
+        public static HashSet<uint> GetQuestsByQuestId(QuestId questId)
```

Comment:
perhaps this method should be renamed so its more clear that it returns quest schedule ids

Reply:
Hmm, it's implied that it is a set of QuestScheduleIds since we are getting quests that match a given quest id. I suppose someone not familiar with quest terminology might not catch on that that though.

If going fully verbose `GetMatchingQuestScheduleIdsForQuestId`? Or do you have another suggestion?

Reply:
sounds good to me, thanks